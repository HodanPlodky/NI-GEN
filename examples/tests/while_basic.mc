int main() {
    int sum = 0;
    int i = 0;
    while (i < 1000) {
        int j = 0;
        while (j < 1000) {
            sum = sum + i + j;
            j = j + 1;
        }
        i = i + 1;
    }
    return sum;
}
