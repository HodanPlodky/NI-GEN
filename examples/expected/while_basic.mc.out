input file : examples/tests/while_basic.mc
return 192
